src/HealthChecks.Rabbitmq/DependencyInjection/RabbitMQHealthCheckBuilderExtensions.cs

```diff
@@ -165,80 +180,19 @@ public static class RabbitMQHealthCheckBuilderExtensions
     /// <returns>The specified <paramref name="builder"/>.</returns>
     public static IHealthChecksBuilder AddRabbitMQ(
         this IHealthChecksBuilder builder,
-        Func<IServiceProvider, IConnectionFactory> connectionFactoryFactory,
+        Func<IServiceProvider, RabbitMQHealthCheckOptions>? setup,
```

Comment:
Let's change `Func<IServiceProvider, RabbitMQHealthCheckOptions>` to `Action<IServiceProvider, RabbitMQHealthCheckOptions>`.

Reply:
Thanks, it's been changed